index.html

```diff
@@ -0,0 +1,350 @@
+            </div>
+          </section>
+          <section class="pricing-tables">
+            <h4 class="pricing-header white-color">STANDARD</h4>
+            <ul>
+              <li class="pricing-plan"><span class="pricing-cost">$10.99</span> /MONTH</li>
+              <li class="pricing-stats">7 Projects</li>
+              <li class="pricing-stats">5 Users</li>
+              <li class="pricing-stats">10 Gb User</li>
+              <li class="pricing-stats">5 Calendar</li>
+            </ul>
+            <div class="pricing-button-container">
+              <a href=""><button class="pricing-button">SIGN UP</button></a>
+            </div>
+          </section>
+          <section class="pricing-tables">
+            <h4 class="pricing-header white-color">PROFESSIONAL</h4>
+            <ul>
+              <li class="pricing-plan"><span class="pricing-cost">$25.99</span> /MONTH</li>
+              <li class="pricing-stats">15 Projects</li>
+              <li class="pricing-stats">50 Users</li>
+              <li class="pricing-stats">100 Gb User</li>
+              <li class="pricing-stats">25 Calendar</li>
+            </ul>
+            <div class="pricing-button-container">
+              <a href=""><button class="pricing-button">SIGN UP</button></a>
+            </div>
+          </section>
+          <section class="pricing-tables">
+            <h4 class="pricing-header white-color">PREMIUM</h4>
+            <ul>
+              <li class="pricing-plan"><span class="pricing-cost">$49.99</span> /MONTH</li>
+              <li class="pricing-stats">UNLIMITED Projects</li>
+              <li class="pricing-stats">UNLIMITED Users</li>
+              <li class="pricing-stats">1000 Gb User</li>
+              <li class="pricing-stats">UNLIMITED Calendar</li>
+            </ul>
+            <div class="pricing-button-container">
+              <a href=""><button class="pricing-button">SIGN UP</button></a>
+            </div>
+          </section>
+        </section>
+      </article>
+      <article id="portfolio" class="portfolio-container section-arrows-orange">
+        <h2 class="portfolio-section-header gray-line">PORTFOLIO</h2>
+        <div class="portfolio-slider">
+          <figure class="portfolio-slider-figure">
+            <img class="portfolio-slider-overlay" src="/img/slider.jpg" width="940" height="336" alt="image advertising Crispers iphone app" title="Crispers/Clouders app">
+            <figcaption class="portfolio-slider-caption">
+              <h3 class="white-color">Crispers</h3>
+              <p>Nice, crispy and creative landing page</p>
+              <a href="http://google.com"><button class="portfolio-slider-button">GET THE APP</button></a>
+              <p>Available for iPhone and iPad</p>
+            </figcaption>
+          </figure>
+          <ul class="portfolio-slider-bullets" title="slider bullets" aria-hidden="true">
+            <li title="first bullet"></li>
```

Comment:
This is a clever solution, but I think it'd end up causing you trouble when you go to implement the sliding functionality.

Rather than having these as empty `li`s, I'd like to see a child element of `a` in each `li` with content in it (a `&bullet;` element is OK, though not very semantic; you could get clever and place a title for each image there and then hide the text while showing a bullet like you've done with the `li` elements, for example). The anchor should have an `href` attribute that links to the appropriate image -- if you go this route, you'll be able to support most javascript sliders, but you could also potentially build up the slider using CSS only (think of the demo of the modal that was displayed using just CSS).